www/i18n/en.json

```diff
@@ -410,10 +282,6 @@
             }
         },
         "permissions": {
-            "locationPermExplanation-android-lt-6": "you accepted the permission during installation. You don't need to do anything now.",
-            "locationPermExplanation-android-6-9": "please select 'allow'",
-            "locationPermExplanation-android-10": "please select 'Allow all the time'",
-            "locationPermExplanation-android-gte-11": "please select 'Allow all the time' for location permissions on the app page",
```

Comment:
I could not find them used in this version of `permissioncheck.js` where the rest of the text for the permission pages is set. Is there another place where it would be handled? If they should be handled with all the other text, I can put these back in the translation files and incorporate them in my rewrite of the permissions page

Reply:
~Here's where they are.~ Here's where I *thought* they were 😄 

https://github.com/e-mission/e-mission-phone/blob/b6da3bbaa1b14f19937c079bef975dc6752c427e/www/js/appstatus/permissioncheck.js#L186

Do we really have two copies of the same text? Our i18n is even more overstuffed than I thought!